### PR TITLE
docs(odin-html): tables.md Turkish translation added

### PR DESCRIPTION
**Pull request by Doguhannilt**
# Description

Translation of tables.md into the Turkish language

PATH OF THE CHANGES: content/odin/intermediate_html_css/intermediate_html_concepts/tables.md

Note: I couldn't find my tag name in Discord

### Checklist

- [ x ] discord username: `doguhan_i`
- [ x] Closes #<issue-number>
- [ ] PR must be created for an issue from issues under "In progress" column from [our project board](https://github.com/orgs/kamp-us/projects/2/views/1).
- [x ] A descriptive and understandable title: The PR title should clearly describe the nature and purpose of the changes. The PR title should be the first thing displayed when the PR is opened. And it should follow the semantic commit rules, and should include the app/package/service name in the title. For example, a title like "docs(@kampus-apps/pano): Add README.md" can be used.
- [x ] Related file selection: Only relevant files should be touched and no other files should be affected.
- [x ] I ran `npx turbo run` at the root of the repository, and build was successful.
- [ ] I installed the npm packages using `npm install --save-exact <package>` so my package is pinned to a specific npm version. Leave empty if no package was installed. Leave empty if no package was installed with this PR.

### How were these changes tested?

Please describe the tests you did to test the changes you made. Please also specify your test configuration.

### CODE REVIEW

File: content/odin/intermediate_html_css/intermediate_html_concepts/tables.md [hunk 1]
```diff
@@ -1,48 +1,46 @@
...
+Bu alanda içerikle alakalı faydalı linkler bulunmaktadır. Zorunlu değildir, ek olarak düşünülmelidir.
 
--   Pencil & Paper published a great [article about things you can achieve with tables](https://pencilandpaper.io/articles/ux-pattern-analysis-enterprise-data-tables) if you dare to spend some time giving them some love. They suggest some good habits on how you should format your data which is pretty simple and makes a huge difference. These are not a list of must dos to a table, but ideas worth keeping in mind for the next time a table is bothering you with how it looks. 
+- Eğer onlara biraz sevgi vererek biraz zaman ayırmaya cesaretin varsa, Pencil & Paper harika bir [article about things you can achieve with tables](https://pencilandpaper.io/articles/ux-pattern-analysis-enterprise-data-tables) yayınladı.  Verilerinizi nasıl biçimlendirmeniz gerektiği konusunda basit ancak büyük farklar yaratabilecek iyi alışkanlıklar öneriyorlar. Bu, bir tabloya karşı bir sonraki sefer sizi rahatsız ediyorsa akılda bulundurulmaya değer fikirlerdir; bunlar bir tablo için mutlaka yapılması gerekenlerin bir listesi değil, ancak düşünmeye değer fikirlerdir.
```
Comment (cansirin) [inline content/odin/intermediate_html_css/intermediate_html_concepts/tables.md:46]:
```suggestion
- Eğer onlara biraz sevgi vererek biraz zaman ayırmaya cesaretin varsa, Pencil & Paper harika bir [tablolarla alakalı makale](https://pencilandpaper.io/articles/ux-pattern-analysis-enterprise-data-tables) yayınladı.  Verilerinizi nasıl biçimlendirmeniz gerektiği konusunda basit ancak büyük farklar yaratabilecek iyi alışkanlıklar öneriyorlar. Bu, bir tabloya karşı bir sonraki sefer sizi rahatsız ediyorsa akılda bulundurulmaya değer fikirlerdir; bunlar bir tablo için mutlaka yapılması gerekenlerin bir listesi değil, ancak düşünmeye değer fikirlerdir.
```

File: content/odin/intermediate_html_css/intermediate_html_concepts/tables.md [hunk 2]
```diff
@@ -1,48 +1,46 @@
...
+1. MDN'den [Tables Basics](https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Basics) ve [Tables Advanced](https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Advanced) öğreticilerini okuyunuz. Bu makaleler HTML tablolarının arkasında olan tüm sözdizimini öğretecektir ve fazlasıyla kolaydır. Okuduğunuz makalelerle birlikte kodladığınızdan emin olun!
+2. MDN'den [Tables Assessment](https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Structuring_planet_data) yapınız. Yeni öğrendiğiniz becerileri pratiğe dökmek güzel!
 
 </div>
 
-### Knowledge check
+### Bilgi ölçme
 
-This section contains questions for you to check your understanding of this lesson on your own. If you’re having trouble answering a question, click it and review the material it links to.
+Bu bölümde, bu dersi anladığınızı kendi başınıza kontrol etmeniz için sorular bulunmaktadır. Eğer ki bir soruyu cevaplandırmakta bir sıkıntı çekerseniz, tıklayın ve bağlantı verdiği materyali inceleyin.
```
Comment (cansirin) [inline content/odin/intermediate_html_css/intermediate_html_concepts/tables.md:35]:
```suggestion
Bu bölümde, bu dersi anladığınızı kendi başınıza kontrol etmeniz için sorular bulunmaktadır. Eğer ki bir soruyu cevaplandırmakta bir sıkıntı çekerseniz, tıklayın ve bağlantıdaki materyali inceleyin.
```

File: content/odin/intermediate_html_css/intermediate_html_concepts/tables.md [hunk 3]
```diff
@@ -1,48 +1,46 @@
-### Introduction
+### Giriş
```
Comment (hueypov) [inline content/odin/intermediate_html_css/intermediate_html_concepts/tables.md:1]:
```suggestion
---
title: 'Tablolar'
---

### Giriş
```

**Comment by gulech** [comment]
Ellerine sağlık Doguy, çok teşekkürler. #738 'deki PR'da tables dosyasını da eklemişsin o yüzden bu PR'ı kapatıyorum.

**Pull request closed without merging**